editions/2023/fa/0x01-about-owasp.md

```diff
@@ -0,0 +1,66 @@
+# <div dir="rtl">درباره OWASP </div>
+
+<div dir="rtl">
+پروژه بازمتن امنیت وب اپلیکیشن‌ها (OWASP)  جامعه ای باز و آزاد است که اختصاصا در حوزه توانمندسازی سازمان‌ها در حوزه توسعه، تهیه و ایجاد اپلیکیشن‌ها و APIهای قابل اعتماد فعالیت دارد.
+ </div>
+<div dir="rtl">
+ در OWASP، موارد زیر را بصورت رایگان و آزاد خواهید یافت:
+</div>
+
+<div dir="rtl">
+<ul>
+  <li>استانداردها و ابزارهای امنیت اپلیکیشن.</li>
+  <li>کتاب‌هایی درباره تست امنیت اپلیکیشن‌ها، توسعه ایمن کد و بازبینی امنیت کد.</li>
+  <li>ارائه‌ها و <a href="https://www.youtube.com/user/OWASPGLOBAL">ویدئوها</a>.</li>
+  <li><a href="https://owasp.org/index.php/OWASP_Cheat_Sheet_Series">راهنما و برگه تقلب</a> برای بسیاری از موضوعات رایج.</li>
+  <li>کنترل‌ها و کتابخانه‌های استاندارد در حوزه امنیت.</li>
+  <li><a href="https://owasp.org/index.php/OWASP_Chapter">شعب محلی در سرتاسر جهان</a>.</li>
+  <li>تحقیقات به روز و پیشرو در حوزه امنیت.</li>
+  <li><a href="https://owasp.org/index.php/Category:OWASP_AppSec_Conference">کنفرانس‌های تخصصی</a> در سرتاسر جهان.</li>
+  <li><a href="https://lists.owasp.org/mailman/listinfo">یست‌های پست الکترونیک</a> برای ارسال اخبار.</li>
+</ul>
```

Comment:
I don't see any special requirement to use Markup instead of Markdown [as in the original file](https://raw.githubusercontent.com/OWASP/API-Security/refs/heads/master/editions/2023/en/0x01-about-owasp.md).

Please carefully review the translation source for issues like this one.